packages/address-book/src/address-book/base/tokens/tokens.ts

```diff
@@ -2930,4 +2930,120 @@ export const tokens = {
+    documentation: 'https://docs.metacade.co/',
+    bridge: 'base-canonical',
+  },
+  SPECTRA: {
+    name: 'Spectra Token',
+    symbol: 'SPECTRA',
+    oracleId: 'SPECTRA',
+    address: '0x64FCC3A02eeEba05Ef701b7eed066c6ebD5d4E51',
+    chainId: 8453,
+    decimals: 18,
+    logoURI: '',
+    website: 'https://spectra.finance/',
+    description:
+      'Spectra is an EVM-centric protocol for interest rate derivatives with an easy-to-use flagship app. The Spectra protocol is permissionless, meaning its services are entirely open for public use. Anyone can create new markets at will, swap yield derivatives, or become a liquidity provider.',
+    documentation: 'https://docs.spectra.finance/',
+    bridge: 'layer-zero',
+    risks: ['NO_TIMELOCK', 'LARGE_HOLDERS'],
+  },
+  SQD: {
+    name: 'Subsquid',
+    symbol: 'SQD',
+    oracleId: 'SQD',
+    address: '0xd4554BEa546EFa83C1E6B389ecac40EA999B3E78',
+    chainId: 8453,
+    decimals: 18,
+    logoURI: '',
+    website: 'https://www.sqd.ai/',
+    description:
+      'SQD Network is a decentralized query engine optimized for batch extraction of large volumes of data. It currently serves historical on-chain data ingested from 200+ EVM and Substrate networks, as well as Solana (in beta), Tron, Starknet and Fuel.',
+    documentation: 'https://docs.sqd.ai/',
+    bridge: 'base-canonical',
+    risks: ['NO_TIMELOCK', 'LARGE_HOLDERS'],
+  },
+  TLOS: {
+    name: 'TLOS',
+    symbol: 'TLOS',
+    oracleId: 'TLOS',
+    address: '0x7252c865c05378Ffc15120F428dd65804dD0CE63',
+    chainId: 8453,
+    decimals: 18,
+    logoURI: '',
+    website: 'https://www.telos.net/',
+    description:
+      'Telos is a next-generation Delegated Proof of Stake (DPOS) Layer-1 blockchain network. It features unsurpassed compatibility for developers and provides an extensive suite of tools and onboarding support to develop and deploy smart contracts on the worlds fastest, most scalable, and affordable Telos EVM',
+    documentation: 'https://docs.telos.net/',
+    bridge: 'layer-zero',
+    risks: ['NO_TIMELOCK'],
+  },
+  AWE: {
+    name: 'AWE Network',
+    symbol: 'AWE',
+    oracleId: 'AWE',
+    address: '0x1B4617734C43F6159F3a70b7E06d883647512778',
+    chainId: 8453,
+    decimals: 18,
+    logoURI: '',
+    website: 'https://www.awenetwork.ai/',
+    description:
+      'AWE Network is opening the portal to Autonomous Worlds where AI Agents collaborate, adapt and evolve. The Autonomous Worlds Engine (AWE) is a modular framework enabling the creation of self-sustaining worlds for scalable agent-agent and human-agent collaboration.',
+    documentation: 'https://docs.awenetwork.ai/',
+    risks: ['NO_TIMELOCK', 'LARGE_HOLDERS'],
+  },
+  CXT: {
```

Comment:
CXT seems to be mintable by multisig on eth?
https://etherscan.io/address/0x7ABc8A5768E6bE61A6c693a6e4EAcb5B60602C4D